qa/L0_lint/pylintrc

```diff
@@ -23,7 +23,8 @@ disable=too-many-locals,
         global-statement,
         too-many-branches,
         global-variable-not-assigned,
-        redefined-argument-from-local
+        redefined-argument-from-local,
+        too-many-function-args

```

Comment:
Why is this needed? Tests seem to pass w/o this

Reply:
This is for some of the autograd functions. For some you are working around this by passing `*args` lists, but not all have that treatment (and frankly it is not really any better than just passing the arguments directly).

Reply:
The `*args` is to handle different arguments for the onnx case, not to circumvent the linting problem.

Reply:
Okay, so if you look at the linter logs for when it threw an error for `too-many-function-args`, it was before your [Fix FP8 paths](https://github.com/NVIDIA/TransformerEngine/pull/67/commits/c36f3fbd2e031b212e545e93f39be0fb59d9f0c2) commit. So it was actually a useful message because the function was being incorrectly called with too many arguments.

Reply:
So we should remove from the `disable` list because it catches actual bugs.

Reply:
My bad, I thought about a different pylint warning. Reenabling.